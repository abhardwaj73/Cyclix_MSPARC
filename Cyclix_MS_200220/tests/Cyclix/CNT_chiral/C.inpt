CELL: 24.000000000000000 0.698131700797732 1.523592733066272
FD_GRID: 140 70 15 #(h=0.2)
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0.0 0.0 0.0
BC: d c h
TWIST_ANGLE: 0.163647899385235
NSTATES: 6
CHEB_DEGREE: 80

EXCHANGE_CORRELATION: LDA_PW
TOL_SCF: 1e-6
TOL_POISSON: 1e-8
TOL_PSEUDOCHARGE: 1e-7
SMEARING: 0.001
ELEC_TEMP_TYPE: fd
TOL_LANCZOS: 1.00E-02
CALC_STRESS: 1
