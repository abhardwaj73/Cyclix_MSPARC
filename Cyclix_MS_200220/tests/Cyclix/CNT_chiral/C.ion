#=========================
# format of ion file
#=========================
# ATOM_TYPE: <atom type name> <valence charge>
# N_TYPE_ATOM: <num of atoms of this type>
# COORD:
# <xcoord> <ycoord> <zcoord>
# ...
# RELAX:
# <xrelax> <yrelax> <zrelax>
# ...


# Reminder: when changing number of atoms, change the RELAX flags accordingly
#           as well.

ATOM_TYPE: C               # atom type followed with valence charge
N_TYPE_ATOM: 2            # number of atoms of this type
PSEUDO_POT: C.psp8 
COORD_FRAC:                # coordinates follows
   0.735000611269835                   0                   0
   0.735000611269835   0.333333333333333   0.666666666666667


