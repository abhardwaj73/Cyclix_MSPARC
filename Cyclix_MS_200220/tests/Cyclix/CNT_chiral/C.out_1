***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 30-Dec-2021  Start time: 00:14:05                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 24.000000 0.698132 1.523593 
TWIST_ANGLE: 0.163648 
FD_GRID: 120 61 8
FD_ORDER: 12
BC: D C H
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 80
NSTATES: 6
EXCHANGE_CORRELATION: LDA_PW
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Cyclix/CNT_chiral/C.out_1
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.200000 (Bohr)
Mesh spacing in y-direction        :  0.201886 (Bohr)
Mesh spacing in z direction        :  0.190449 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/Cyclix/CNT_chiral/C.out_1
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.69 1.06 7.62
Number of atoms of type 1          :  2
Estimated total memory usage       :  175.70 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.3595606016E+00        4.425E-01        18.851
2            -5.4857197786E+00        3.726E-01        11.897
3            -5.5012239949E+00        3.022E-01        7.357
4            -5.5066482788E+00        2.045E-01        9.242
5            -5.5108779083E+00        2.167E-02        9.429
6            -5.5108764034E+00        8.701E-03        7.162
7            -5.5109341916E+00        5.091E-03        7.924
8            -5.5109316910E+00        2.050E-03        6.112
9            -5.5109468038E+00        5.293E-04        5.611
10           -5.5109400914E+00        1.758E-04        5.871
11           -5.5109424817E+00        1.205E-04        4.057
12           -5.5109423830E+00        3.314E-05        5.076
13           -5.5109420342E+00        1.635E-05        3.519
14           -5.5109422471E+00        2.898E-06        2.620
15           -5.5109421736E+00        2.044E-06        2.594
16           -5.5109421236E+00        1.144E-06        2.722
17           -5.5109421367E+00        8.244E-07        2.393
Total number of SCF: 17    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.5109421367E+00 (Ha/atom)
Total free energy                  : -1.1021884273E+01 (Ha)
Band structure energy              : -3.6089330842E+00 (Ha)
Exchange correlation energy        : -3.5128987342E+00 (Ha)
Self and correction energy         : -3.8575730745E+01 (Ha)
Entropy*kb*T                       : -2.5696989515E-09 (Ha)
Fermi level                        : -6.4334292075E-02 (Ha)
Average force                      :  4.4122461535E-01 (Ha/Bohr)
Maximum force                      :  4.4136691492E-01 (Ha/Bohr)
Time for force calculation         :  3.827 (sec)
Stress (Ha/Bohr)                   :
  9.2797254034E+01 
Stress equiv. all periodic (GPa)   :
  1.0263671295E+03 
Max stress                         :  9.2797254034E+01 (Ha/Bohr)
Max stress equiv. all periodic     :  1.0263671295E+03 (GPa)
Time for stress calculation        :  10.089 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  151.121 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
