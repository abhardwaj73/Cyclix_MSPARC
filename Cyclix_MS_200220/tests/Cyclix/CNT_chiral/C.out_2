***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 30-Dec-2021  Start time: 00:26:01                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 24.000000 0.698132 1.523593 
TWIST_ANGLE: 0.163648 
FD_GRID: 140 70 15
FD_ORDER: 12
BC: D C H
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 80
NSTATES: 6
EXCHANGE_CORRELATION: LDA_PW
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Cyclix/CNT_chiral/C.out_2
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.171429 (Bohr)
Mesh spacing in y-direction        :  0.175929 (Bohr)
Mesh spacing in z direction        :  0.101573 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/Cyclix/CNT_chiral/C.out_2
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.64 1.06 7.62
Number of atoms of type 1          :  2
Estimated total memory usage       :  440.52 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.5063951529E+00        1.299E-01        56.295
2            -5.5798767626E+00        1.274E-01        52.572
3            -5.5869869409E+00        9.978E-02        36.253
4            -5.5911890255E+00        3.343E-02        38.270
5            -5.5918213793E+00        7.703E-03        37.342
6            -5.5919118692E+00        5.921E-03        35.204
7            -5.5919131520E+00        2.553E-03        32.518
8            -5.5919190002E+00        1.250E-03        25.914
9            -5.5919221670E+00        4.424E-04        27.252
10           -5.5919205821E+00        2.724E-04        18.433
11           -5.5919213406E+00        1.328E-04        19.435
12           -5.5919209788E+00        4.557E-05        18.859
13           -5.5919211294E+00        7.975E-06        12.625
14           -5.5919210906E+00        3.160E-06        9.491
15           -5.5919211067E+00        1.363E-06        7.392
16           -5.5919211067E+00        7.464E-07        6.786
Total number of SCF: 16    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.5919211067E+00 (Ha/atom)
Total free energy                  : -1.1183842213E+01 (Ha)
Band structure energy              : -3.7083688662E+00 (Ha)
Exchange correlation energy        : -3.5265671249E+00 (Ha)
Self and correction energy         : -3.3543640766E+01 (Ha)
Entropy*kb*T                       : -8.4823018028E-10 (Ha)
Fermi level                        : -6.5157564064E-02 (Ha)
Average force                      :  3.2611369064E-03 (Ha/Bohr)
Maximum force                      :  3.2773258927E-03 (Ha/Bohr)
Time for force calculation         :  9.544 (sec)
Stress (Ha/Bohr)                   :
 -3.2144424027E-01 
Stress equiv. all periodic (GPa)   :
 -3.5552754834E+00 
Max stress                         :  3.2144424027E-01 (Ha/Bohr)
Max stress equiv. all periodic     :  3.5552754834E+00 (GPa)
Time for stress calculation        :  19.077 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  530.169 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
