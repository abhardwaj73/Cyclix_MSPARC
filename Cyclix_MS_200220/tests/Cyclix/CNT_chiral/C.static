***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of atoms:
      0.7350006113       0.0000000000       0.0000000000
      0.7350006113       0.3333333333       0.6666666667
Atomic forces (Ha/Bohr):
     -0.0247814954       0.0600483496       0.4368056851
      0.0004884614      -0.0649640564      -0.4368056851
