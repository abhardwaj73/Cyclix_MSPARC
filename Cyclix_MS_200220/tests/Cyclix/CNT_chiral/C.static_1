***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of atoms:
      0.7350006113       0.0000000000       0.0000000000
      0.7350006113       0.3333333333       0.6666666667
Atomic forces (Ha/Bohr):
     -0.0246210039       0.0583058317       0.4368054369
     -0.0003799249      -0.0612737726      -0.4368054369
