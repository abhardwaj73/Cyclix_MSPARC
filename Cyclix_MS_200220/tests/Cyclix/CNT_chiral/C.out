***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 20-Feb-2020  Start time: 15:17:30                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 24.000000 0.698132 1.523593 
TWIST_ANGLE: 0.163648 
FD_GRID: 120 61 8
FD_ORDER: 12
BC: D C H
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 80
NSTATES: 6
EXCHANGE_CORRELATION: LDA_PW
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/Cyclix/CNT_chiral/C.out_1
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.200000 (Bohr)
Mesh spacing in y-direction        :  0.201886 (Bohr)
Mesh spacing in z direction        :  0.190449 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/Cyclix/CNT_chiral/C.out_1
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  C 4
Pseudopotential                    :  C.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.55 1.43 6.48
Number of atoms of type 1          :  2
Estimated total memory usage       :  175.70 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.3595607564E+00        4.425E-01        8.677
2            -5.4857198511E+00        3.726E-01        10.477
3            -5.5012240578E+00        3.022E-01        11.949
4            -5.5066483412E+00        2.045E-01        7.471
5            -5.5108779623E+00        2.167E-02        9.350
6            -5.5108764632E+00        8.701E-03        9.442
7            -5.5109342494E+00        5.091E-03        7.069
8            -5.5109317482E+00        2.049E-03        7.929
9            -5.5109468622E+00        5.293E-04        6.192
10           -5.5109401500E+00        1.759E-04        5.703
11           -5.5109425387E+00        1.207E-04        5.933
12           -5.5109424414E+00        3.304E-05        4.121
13           -5.5109420931E+00        1.638E-05        5.179
14           -5.5109423053E+00        2.893E-06        3.596
15           -5.5109422314E+00        2.049E-06        2.679
16           -5.5109421820E+00        1.132E-06        2.656
17           -5.5109421950E+00        8.161E-07        2.774
Total number of SCF: 17    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -5.5109421950E+00 (Ha/atom)
Total free energy                  : -1.1021884390E+01 (Ha)
Band structure energy              : -3.6089324225E+00 (Ha)
Exchange correlation energy        : -3.5128986597E+00 (Ha)
Self and correction energy         : -3.8575730747E+01 (Ha)
Entropy*kb*T                       : -1.2953158613E-09 (Ha)
Fermi level                        : -1.7992473676E-01 (Ha)
Average force                      :  4.4161006961E-01 (Ha/Bohr)
Maximum force                      :  4.4161043216E-01 (Ha/Bohr)
Time for force calculation         :  3.406 (sec)
Stress (Ha/Bohr)                   :
  9.2797355375E+01 
Stress equiv. all periodic (GPa)   :
  1.0263682504E+03 
Max stress                         :  9.2797355375E+01 (Ha/Bohr)
Max stress equiv. all periodic     :  1.0263682504E+03 (GPa)
Time for stress calculation        :  10.542 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  154.717 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
